### PR TITLE
[bitmarket] new features

### PR DESCRIPTION
**Pull request by npomfret**

### CODE REVIEW

**Comment by npomfret** [comment]
Pls squash

File: js/bitmarket.js [hunk 1]
```diff
@@ -185,6 +180,96 @@ module.exports = class bitmarket extends Exchange {
...
+        //         received_in: '1EdAqY4cqHoJGAgNfUFER7yZpg1Jc9DUa3',
+        //     currency: 'BTC',
+        //     amount: 0.49926113,
+        //     time: 1518353534,
+        //     commission: 0.0008,
+        //     withdraw_type: 'Cryptocurrency' }
+        const ccyCode = this.safeString (item, 'currency');
+        const time = this.safeInteger (item, 'time');
+        const timestamp = time * 1000;
+        const ccyId = this.currencyId (ccyCode);
```
Comment (kroitor) [inline js/bitmarket.js]: `currencyId` gets a unified currency code and returns an currency id...
In this case we need the opposite – to convert from a currency id, to a common currency code, which is usually done in the following way:

```Javascript
let code = undefined;
const currencyId = this.safeString (item, 'currency');
if (currencyId !== undefined) {
    if (currencyId in this.currencies_by_id) {
        code = this.currencies_by_id[currencyId]['code'];
    } else {
        code = this.commonCurrencyCode (currencyId);
    }
}
```

Comment (kroitor) [reply js/bitmarket.js]: It just so happens that for this exchange codes are equal to ids, so the bug does not show up, but still does live here (may break it in the future).

Comment (npomfret) [reply js/bitmarket.js]: Opps, thanks.

**Pull request merged**